subsys/bluetooth/mesh/settings.c

```diff
@@ -1354,6 +1413,44 @@ static void store_pending_mod(struct bt_mesh_model *mod,
 	}
 }
 
+#define IS_VA_DEL(_label)	((_label)->ref == 0)
+static void store_pending_va(void)
+{
+	struct label *lab;
+	struct va_val va;
+	char path[18];
+	u16_t i;
+	int err;
+
+	for (i = 0; (lab = get_label(i)) != NULL; i++) {
+		if (atomic_test_and_clear_bit(lab->flag,
+					BT_MESH_VA_CHANGED)) {
```

Comment:
Make this:
```
if (!atomic_test_and_clear_bit(...)) {
        continue;
}

...
```

That way you reduce one level of indentation from most of the stuff in the for-loop and avoid having to do line splits so easily.